src/cache-manager.js

```diff
@@ -150,6 +152,9 @@ class CacheManager {
         await this.insertCacheEntry(rawUrl, 'present', Math.floor(setTTL));
         this.debug(`backfilled cache for ${rawUrl}`);
         outcome.status = 'present';
+
+        this.monitor.count(`id:${this.id}.backfill`, 1);
```

Comment:
Why `id:` how about `id.<id>` or maybe there is a better name than id... like `storage.<id>` or something...